### DISCUSSION TITLE
Troubleshoot - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5004] [Announcements]
# Troubleshoot - DaoCloud Enterprise

DaoCloud Enterprise 5.0 is the next-generation PaaS platform, created with many leading cloud-native technologies, to easily promote and boost your digital business.

https://docs.daocloud.io/en/network/modules/calico/debug/

**Comment by samuelerl** [comment]
Hello,
Question regarding vxlan.calico status. I have a server that has vxlan.calico interface in an unknown state always. I've tried researching why this is and the impact it has on service - but all my pods are running and I was unable to find a reason. You state that this is expected behavior, but I am unable to confirm on any other source that this is such. Could you please confirm whether or not the interface should be in an unknown state?

**Reply by cyclinder** [reply]
Hello @samuelerl, sorry for the slow response, 

> "state UNKNOWN" isn't the administrative "up/down" switch, it's the operational state that comes from the system itself shows a whole selection of values. This state needs to be provided by the driver – and (at least according to Linux documentation) not all drivers have the necessary bits to set the actual oper state. So if your Ethernet driver hasn't done so, you will see the default value IF_OPER_UNKNOWN (0).

If you try to up the `vxlan.calico`, you will see the device still in UNKNOWN state, It's the desired state, maybe we shouldn't emphasize its status here.
